tools/clang/unittests/HLSL/ShaderOpTest.cpp

```diff
@@ -902,7 +902,12 @@ void ShaderOpTest::RunCommandList() {
 
 #if defined(NTDDI_WIN10_VB) && WDK_NTDDI_VERSION >= NTDDI_WIN10_VB
     if (m_pShaderOp->MS) {
-      ID3D12GraphicsCommandList6 *pList6 = m_CommandList.List.p;
+#ifndef NDEBUG
+      D3D12_FEATURE_DATA_D3D12_OPTIONS7 O7;
+      DXASSERT_LOCALVAR(O7, SUCCEEDED(m_pDevice->CheckFeatureSupport((D3D12_FEATURE)D3D12_FEATURE_D3D12_OPTIONS7, &O7, sizeof(O7))), "mesh shader test enabled on platform without mesh support");
+#endif
+      ID3D12GraphicsCommandList6 *pList6;
```

Comment:
Should be a `CComPtr` probably, since `QueryInterface` adds a reference. Otherwise this will leak.